cmd/cluster-olm-operator/main.go

```diff
@@ -369,3 +182,105 @@ func replaceImageHook(placeholder string, desiredImageEnvVar string) deploymentc
 		return []byte(newDeployment), nil
 	}
 }
+
+type controllerBuilder struct {
```

Comment:
Doesn't necessarily need to be in this PR, but I think it's time to split this out into a separate file and possibly package.